test/integration_testcase.go

```diff
@@ -78,6 +78,9 @@ func (tc *IntegrationTestCase) Cleanup() {
 		j = bytes.Replace(j, cmds, n, -1)
 		j = append(j, '\n')
 		err = ioutil.WriteFile(filepath.Join(tc.rootPath, "testcase.json"), j, 0666)
+		if err != nil {
+			tc.t.Logf("Failed to update testcase %s: %s", tc.name, err)
+		}
```

Comment:
If writing the updated golden file fails, we should at least log the error. Should we complain more loudly than that?